TwilioVerifySDK/TwilioVerify/Sources/Extensions/URLSession+Extensions.swift

```diff
@@ -48,8 +48,19 @@ extension URLSession {
   }
 }
 
-internal struct FailureResponse {
-  let responseCode: Int
-  let errorData: Data
-  let headers: [AnyHashable: Any]
+public struct ServerErrorResponse: Codable {
```

Comment:
Checking [this page](https://www.twilio.com/docs/api/errors), it would be better to change this to `APIError`, wdyt?

Reply:
Sounds good, I just changed